cinema_service/settings.py

```diff
@@ -86,8 +91,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
```

Comment:
It's great that you're loading your database configuration from environment variables. This is a good practice for maintaining the security of your application.